### ISSUE TITLE
onnx.onnx_cpp2py_export.checker.ValidationError: Node () has input size 1 not in range [min=2, max=3].

### ISSUE THREAD

**Issue by zhonhel** [onnx/onnx#2548]
Hi all~

I'm converting a pytorch model to an onnx model.
The convert process doesn't report any error or warning.
But when I check the onnx model, using this function:
```
onnx.checker.check_model(model)
```
I get the following error:
```
onnx.onnx_cpp2py_export.checker.ValidationError: Node () has input size 1 not in range [min=2, max=3].

==> Context: Bad node spec: input: "626" output: "634" op_type: "Pad" attribute { name: "mode" s: "constant" type: STRING } attribute { name: "pads" ints: 0 ints: 0 ints: 1 ints: 1 ints: 0 ints: 0 ints: 1 ints: 1 type: INTS } attribute { name: "value" f: 0 type: FLOAT }
```
Can anyone help me?  Thank you very much!

**Comment by linkerzhang** [comment]
adding @spandantiwari for thoughts.

Looks like the pytorch model's pad is converted into Pad-2, however, the model's opset information is using op set version 11 (Pad-11 needs 2-3 inputs)

**Comment by zhonhel** [comment]
@linkerzhang 
Thank you very much！
I indeed use parameter
```opset_version=11```
in 
```torch.onnx.export```
function.
Because if I use:
```opset_version=9(default)```
it will occur the following error:
```RuntimeError: ONNX export failed: Couldn't export operator aten::upsample_bilinear2d```

**Comment by zhonhel** [comment]
@linkerzhang 
I find the error:
```
RuntimeError: ONNX export failed: Couldn't export operator aten::upsample_bilinear2d
```
is because I use the follow op in pytorch:
```
F.interpolate(x, size=self.out_size, mode="bilinear", align_corners=True)
```
when i change this op to:
```
F.interpolate(x, size=self.out_size, mode="bilinear", align_corners=False)
```
This error is solved.
But when I use mxnet load the generated onnx model using this function: `mxnet.contrib.onnx.import_model`, I get the following error:
```
  File "(/path/to)/miniconda3/lib/python3.7/site-packages/mxnet/contrib/onnx/onnx2mx/import_onnx.py", line 118, in <listcomp>
    inputs = [self._nodes[i] for i in node.input]
KeyError: 'entry.conv1.0.weight'
```

**Comment by linkerzhang** [comment]
have no idea about mxnet, try onnx runtime? :)

**Comment by mrlzla** [comment]
Same problem

**Comment by mrlzla** [comment]
Fixed when upgraded to pytorch 1.4.0
